.ci/azure/android_arm64.yml

```diff
@@ -0,0 +1,152 @@
+    displayName: 'System info'
+
+  - script: |
+      rm -rf $(WORK_DIR)      ; mkdir $(WORK_DIR)
+      rm -rf $(BUILD_DIR)     ; mkdir $(BUILD_DIR)
+      rm -rf $(ANDROID_TOOLS) ; mkdir $(ANDROID_TOOLS)
+    displayName: 'Make dir'
+
+  - checkout: self
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino_contrib
+
+  - checkout: openvino
+    clean: true
+    lfs: false
+    submodules: recursive
+    path: openvino
+
+  - script: |
+      set -e
+      $(OPENVINO_REPO_DIR)/install_build_dependencies.sh
+      # Move into contrib install_build_dependencies.sh
+      sudo apt --assume-yes install scons crossbuild-essential-arm64 libprotoc-dev protobuf-compiler
+      # OpenCV should provide install_build_dependencies.sh as well
+      # Move into resources
+      git clone https://github.com/opencv/opencv.git --depth 1 $(OPENCV_REPO_DIR)
+      # Speed up build
+      wget https://github.com/ninja-build/ninja/releases/download/v1.10.2/ninja-linux.zip
+      unzip ninja-linux.zip
+      sudo cp -v ninja /usr/local/bin/
+      # Install Android SDK, NDK and TOOLS
+      wget https://dl.google.com/android/repository/commandlinetools-linux-7583922_latest.zip
+      unzip commandlinetools-linux-7583922_latest.zip
+      yes | ./cmdline-tools/sdkmanager --sdk_root=$(ANDROID_TOOLS) --licenses
+      ./cmdline-tools/sdkmanager --sdk_root=$(ANDROID_TOOLS) --install "ndk-bundle"
+      ./cmdline-tools/sdkmanager --sdk_root=$(ANDROID_TOOLS) --install "platform-tools"
+      ./cmdline-tools/sdkmanager --sdk_root=$(ANDROID_TOOLS) --install "platforms;android-$(ANDROID_SDK_VERSION)"
+    workingDirectory: $(WORK_DIR)
+    displayName: 'Install dependencies'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DCMAKE_TOOLCHAIN_FILE=$(ANDROID_TOOLS)/ndk-bundle/build/cmake/android.toolchain.cmake
+        -DANDROID_SDK_ROOT=$(ANDROID_TOOLS)/platforms/android-$(ANDROID_SDK_VERSION)
+        -DANDROID_SDK_TOOLS=$(ANDROID_TOOLS)/platform-tools
+        -DBUILD_ANDROID_PROJECTS=OFF
+        -DANDROID_ABI=$(ANDROID_ABI_CONFIG) 
+        -DANDROID_STL=c++_shared
+        -DANDROID_PLATFORM=$(ANDROID_SDK_VERSION)
+        $(OPENCV_REPO_DIR)
+      workingDirectory: $(BUILD_DIR_OPENCV)
+
+  - script: ninja
+    workingDirectory: $(BUILD_DIR_OPENCV)
+    displayName: 'Build OpenCV Android ARM64'
+
+  - script: ninja install
+    workingDirectory: $(BUILD_DIR_OPENCV)
+    displayName: 'Install OpenCV Android ARM64'
+
+  - task: CMake@1
+    inputs:
+      cmakeArgs: >
+        -GNinja
+        -DVERBOSE_BUILD=ON
+        -DCMAKE_BUILD_TYPE=$(BUILD_TYPE)
+        -DCMAKE_TOOLCHAIN_FILE=$(ANDROID_TOOLS)/ndk-bundle/build/cmake/android.toolchain.cmake
+        -DANDROID_ABI=$(ANDROID_ABI_CONFIG)
+        -DANDROID_STL=c++_shared
+        -DANDROID_PLATFORM=$(ANDROID_SDK_VERSION)
+        -DOpenCV_DIR=$(BUILD_DIR_OPENCV)/install/sdk/native/jni
```

Comment:
Why do we need build OpenCV while there are published releases for Android? 

https://github.com/opencv/opencv/releases/download/4.5.5/opencv-4.5.5-android-sdk.zip

Reply:
Thank you, I will correct it near time with other fixes